charts/common/values.yaml

```diff
@@ -0,0 +1,597 @@
+    #nginx.ingress.kubernetes.io/proxy-buffer-size: "16k"
+    # nginx.ingress.kubernetes.io/ssl-redirect needs to be set to 'true' when using SSL/TLS offloading with a LB outside of Kubernetes
+    nginx.ingress.kubernetes.io/ssl-redirect: "true"
+  # tls configuration
+  tls:
+    # define your type of tls certificate, it can be one of:
+    # none: tls will be disabled
+    # existing: use an existing secret already present in the namespace. Requires secret name to be specified
+    # provided: use an officially generated certificate/key
+    # k8s: use the default k8s-ingress tls. no further configuration needed
+    # self: generate a self signed certificate, which is stored as secret. Needs commonName and validityDuration at least
+    type: "none"
+    # depending on the type you have further configuration options:
+    self:
+      # commonName of the certificate (mandatory)
+      commonName: "*.cluster.local"
+      # ipAddresses is an optional list of IP addresses to add in the Subject Alternative Names (SAN) section
+      ipAddresses: []
+      #  - "10.0.0.1"
+      # alternativeDnsNames is an optional list of DNS names to add in the Subject Alternative Names (SAN) sectiom
+      alternativeDnsNames: []
+      #  - "foo.com"
+      # validityDuration defines how long the certificate is valid (in days)
+      validityDuration: 365
+    existing:
+      # name of an existing secret with tls.crt & tls.key content
+      secret: ""
+    provided:
+      # If SSL is terminated on ingress and you have a generated (preferrably CERT-001) certificate/key
+      # Has to be base64 encoded and should be encrypted in the ejson vault
+      # Add Variable to your CI/CD Settings "SKIP_DECRYPT" with value "" that it doesnt decrypt the cert and fails.
+      cert: ""
+      # The key must not have a passphrase
+      key: ""
+# end common.ingress
+
+# start common.servicemonitor
+# servicemonitor is from CoreOS and lets you scrape metrics with the prometheus-operators
+# Hint: The output of the metrics endpoint must be correctly formatted, see https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md
+# - /actuator/metrics in Spring Boot based applications from Bedag, this endpoint is not served (plain json output)
+# - /metrics in Spring Boot based applications, this is the correct endpoint to scrape metrics
+servicemonitor:
+  # deploy has to be set to true for rendering to be applied
+  deploy: false
+  # basicAuth is a dictionary for defining values for setting up basic Authentication
+  basicAuth:
+    # enabled when set to 'true', adds basic authentication to all endpoints
+    enabled: false
+    # existingSecret if not empty (""), points to an existing secret (matching name of the resource)
+    existingSecret: ""
+    # newSecret is a dictionary for defining key/value pairs to be stored in a new secret
+    newSecret: {}
+      #username: bedag
```

Comment:
I would change `username` and `custom-username` to something other than bedag

Reply:
done